lib/rucio/web/rest/flaskapi/v1/rules.py

```diff
@@ -150,14 +150,15 @@ def put(self, rule_id):
             description: No rule found for the given id
        """
         parameters = json_parameters()
-        options = param_get(parameters, 'options')
+        options = param_get(parameters, 'options', default={})
```

Comment:
Doesn’t this require some special mention?

Reply:
i decided to take it out. I want the error for a missing parameter to be raised as early as possible, not later.